packages/cross_file/example/test/readme_excerpts_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cross_file/cross_file.dart';
+import 'package:cross_file_example/readme_excerpts.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
Instructs `flutter test` to only run this in "vm" mode, not in the browser:

```suggestion
import 'package:flutter_test/flutter_test.dart';

@TestOn('vm')

void main() {
```

(Not sure if `@TestOn` can be used from `package:flutter_test` or it's a `package:test` thing, please, adjust the dependency as needed!)